controllers/addons_controller.go

```diff
@@ -292,7 +308,8 @@ func repoMapperFunc(a handler.MapObject) []reconcile.Request {
 }
 
 func indexHelmReleaseByOwner(o runtime.Object) []string {
-	hr, ok := o.(*helmopv1.HelmRelease)
+	reconciler.Log.V(1).Info("processing", "helmreleases.helm.toolkit.fluxcd.io", utils.LogJSON(o))
```

Comment:
This log message is misleading.  Indexing is not the same as processing and logging each indexing call seems likely to be very chatty in the log, especially at the info level.  We might need this information for development purposes, but it should probably be higher than the standard debug verbosity level.

Reply:
done